samples/java-spring-eventsourced-customer-registry/src/it/java/customer/api/CustomerIntegrationTest.java

```diff
@@ -27,26 +26,24 @@
 @SpringBootTest(classes = Main.class)
 public class CustomerIntegrationTest extends KalixIntegrationTestKitSupport {
 
-
   @Autowired
   private WebClient webClient;
 
+  @Autowired
+  private ComponentClient componentClient;
```

Comment:
does that mean that we can wire the `ComponentClient` anywhere? Like wire it into an Entity?

Reply:
No, it's not possible, nothing has changed.